introduction.md

```diff
@@ -0,0 +1 @@
+My name is Neal.  I'm a developer specializing in process and workflow for large development teams.  Since we're looking at adopting github for part of our workflow, I'm running through these training sessions to get a better understanding
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.